### DISCUSSION TITLE
No connection Wazuh agent with Manger

### DISCUSSION THREAD

**Discussion by jozb09** [Security-Onion-Solutions/securityonion#9284] [Unsupported Versions] [answered]
Hello Team,

Sorry, am getting the below error on my endpoint wazuh agent 

ERROR: (1216): Unable to connect to 'XX.XXX.XXX.XXX:1514/tcp': 'No connection could be made because the target machine actively refused it.'. 

Kindly, help me how to fix it. 

The deployment is distributed with three nodes

**Comment by jozb09** [comment]
On Linux
ossec-agentd: INFO: Closing connection to server (XX.XXX.XXX.XXX:1514/udp)

What could be the issue? 

The service under manager node is running

**Comment by dougburks** [comment, marked as answer]
@jozb09 Please do not create duplicate discussions.

https://github.com/Security-Onion-Solutions/securityonion/discussions/9287
